Khan/wonder-blocks
# Fix the autocomplete example for TextField in storybook

## Summary:
The autocomplete example wasn't providing autocompletion options
when you type something in. That's because it was missing the
submit action that a form needs to save this information.

- Put the TextField in a `<form>` and add a submit button.

Issue: https://khanacademy.atlassian.net/browse/WB-1294

<img width="1093" alt="Screen Shot 2022-05-24 at 4 41 47 PM" src="https://user-images.githubusercontent.com/13231763/170149786-3fc7e48e-2c9f-4b7a-a71a-5d6cdffbbda8.png">


## Test plan:
- Go to storybook
- Go to the text field autocomplete example
- type something in
- press the submit button
- refresh the page
- click on the field or start typing something with the same starting letters as the input from before
- Confirm that options show up
